### DISCUSSION TITLE
click cell to view related resource item?

### DISCUSSION THREAD

**Discussion by wivaku** [filamentphp/filament#6065] [Help] [answered]
I have e.g. Orders, and in the table I see order number and the customer name/number.
Would like to click on the customer field in the table to open the Customer view.

Is that possible?

**Comment by danharrin** [comment]
On the column, `->url(fn ($record) => CustomerResource::getUrl('edit', ['record' => $record->customer]))`

**Comment by wivaku** [comment, marked as answer]
Oops... did not catch this in the documentation. Apologies!  
https://filamentphp.com/docs/2.x/tables/columns/getting-started#opening-urls

I ended up using a slight variation, as in my case not every "Order" has a "Customer".

```
                    ->url(function ($record) {
                        return $record->customer
                            ? CustomerResource::getUrl('view', ['record' => $record->customer])
                            : null;
                    })
```

**Comment by mubeen54** [comment]
->url(CompanyResource::getUrl('index'))
that is work for me
